weave/ops_primitives/number_bin.py

```diff
@@ -41,9 +41,11 @@ def body(row):
     render_info={"type": "function"},
 )
 def numbers_bins_equal(arr, bins):
+    arr = [i for i in arr if i != None]
```

Comment:
Should this be `i is not None`?

Reply:
actually, no. The weave engine will often use `BoxedNone` in these cases, which does not work with ref comparison.